### PR TITLE
Apfl nzfl v21

### PR DESCRIPTION
**Pull request by rishikeshkumar9**

### CODE REVIEW

**Review by ppapasani1 — APPROVED** [review]

File: Flood_E2E/VulnCache.R [hunk 1]
```diff
@@ -23,7 +23,11 @@ cghs$HAZ_TYPE= as.numeric(cghs$HAZ_TYPE)
 cghs=data.table(cghs)
 
 #imap = sqlceQuery(ceHandle, "select * From imap")
-imap =read.csv(paste0(Vulndir,"\\Ifm\\CSV\\vimap.csv"),sep = ",",header = TRUE)
+if(cnfg$Model=="APFL" || cnfg$Model=="NZFL"){
```
Comment (sthupakula-rms) [inline Flood_E2E/VulnCache.R:26]: `toupper(cnfg$Model)`

Comment (sthupakula-rms) [reply Flood_E2E/VulnCache.R:26]: Do we really have two variables `config` and `cnfg`, or is there a typo? There's a `config$Version == 15` in the code change above and here there's a `config$Version == 15`

**Review by sthupakula-rms — CHANGES_REQUESTED** [review]
Thank you.
One change request and a question.

File: Flood_E2E/VulnCache.R (hunk 1 above)
Comment (rishikeshkumar9) [reply Flood_E2E/VulnCache.R:26]: cnfg is the variable used when we read config file directly from the path of test folder. The same file is renamed config==cnfg while running hazard or vulnerability . For uniformity i think  "config" should be used for the two changes. I will replace cnfg with config.

**Review by rishikeshkumar9 — COMMENTED** [review]
added to update the APFL version 20

**Pull request merged**